justfile

```diff
@@ -90,22 +90,30 @@ maplibre-native-info: (assert "curl") (assert "jq")
     fi
 
 # Run all tests
-test:
+test-all:
     cargo test --all-targets --workspace
 
+# Run testcases against a specific backend
+test backend="vulkan":
+    cargo test --all-targets --features {{backend}} --workspace
+
 # Run all tests and accept the changes. Requires cargo-insta to be installed.
 test-accept:
     cargo insta test --accept
 
 # Test documentation
-test-doc:
-    RUSTDOCFLAGS="-D warnings" cargo test --doc
-    RUSTDOCFLAGS="-D warnings" cargo doc --no-deps
+test-doc backend="vulcan":
```

Comment:
do note that docs.rs will not know anything about the backend - so we should have a way to compile the documentation without any specific features ... or set some defaults in cargo.toml for docsrs builds

Reply:
yea, I just wanted to be extra carefull. Currently, this is unnessesary.